src/OmniSharp.Dnx/DnxPaths.cs

```diff
@@ -138,31 +132,24 @@ private IEnumerable<string> GetRuntimeLocations()
             {
                 // Newer path
                 yield return Path.Combine(home, ".dnx");
+            }
 
-                // New path
-                yield return Path.Combine(home, ".k");
-
-                // Old path
-                yield return Path.Combine(home, ".kre");
+            if (!string.IsNullOrEmpty(Environment.GetEnvironmentVariable("ProgramFiles")))
```

Comment:
DNVM uses ProgramData or AllUsersProfile and "Microsoft DNX" as the folder name for global dnx
https://github.com/aspnet/dnvm/blob/dev/src/dnvm.ps1#L213-L217